definitions/tools/mark_duplicates_and_sort.cwl

```diff
@@ -10,12 +10,14 @@ requirements:
       coresMin: 8
       ramMin: 40000
     - class: DockerRequirement
-      dockerPull: "mgibio/mark_duplicates-cwl:1.0.0"
+      dockerPull: "zlskidmore/mark_dups_test:latest"
```

Comment:
This should point to `mgibio/mark_duplicates-cwl:1.0.1` after genome/docker-mark_duplicates-cwl#2.

Reply:
```suggestion
      dockerPull: "mgibio/mark_duplicates-cwl:1.0.1"
```

Reply:
@zlskidmore could you make the suggested change by @tmooney?